core/policy/service.go

```diff
@@ -319,7 +319,11 @@ func (s *Service) validateApprover(expr string) error {
 
 	// skip validation if expression is accessing arbitrary value
```

Comment:
can you enhance this comment like this PR's description itself? It'll be more helpful at some point later

Reply:
done